portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -100,6 +100,7 @@ const uploadReportAndSendNotification = async (req, res) => {
     const { file } = req;
 
     const { formattedReportPeriod, reportData, reportPeriod, user } = req.body;
+    // TODO FN-1859 - do we need to parse here?
```

Comment:
most likely, I think this is being sent in formData rather than a normal json body of a request because it's being sent with a file. Either way if you want to check this do and then remove comment please :)

Reply:
I think Mike added this, but wanted to leave it in just so we could have a comment left by it on the PR :)